SpellCheck/pom.xml

```diff
@@ -46,6 +46,24 @@
 			<artifactId>jna</artifactId>
 			<version>3.0.9</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
```

Comment:
We could... I'm going to leave that as a stretch task.  But it might help clean up some of our POMs.